src/cljs/athens/electron/audio.cljs

```diff
@@ -0,0 +1,55 @@
+(ns athens.electron.audio
+  (:require
+    [athens.common.utils :as common.utils]
+    [athens.db :as db]
+    [athens.electron.utils :as electron.utils]
+    [re-frame.core :as rf]))
+
+(defn save-audio
```

Comment:
This and the save-image function share a big portion of code, maybe makes sense to refactor the common code to copy files